templates/component.html

```diff
@@ -2,7 +2,7 @@
   <table border="1">
     <thead>
       <tr>
-        <th>Bug</th><th>Summary</th><th>Component</th><th>Confidence</th>
+        <th>Bug</th><th>Summary</th><th>Component</th><th>Confidence (%)</th>
```

Comment:
I forgot this: if (%) then you must multiply it by 100

Reply:
Done